omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
```

Comment:
It should be below the table. The idea is that the user looks at the table of results (from top to bottom), doesn't see the result they're looking for, and their eyes end up at the bottom with this "didn't find it?" button.